LICENSES/BSD-3-Clause.txt

```diff
@@ -0,0 +1,11 @@
+Copyright (c) <year> <owner>. 
```

Comment:
is this correct?

Reply:
It was added like this be the REUSE tool as it automatically downloads and puts the licenses into this folder.

But I guess we should adapt it by hand then... Let me push the change.

Reply:
Fixed